src/Controller/ContactController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
```

Comment:
Remove use statement for Route class. We do not use route annotations in this class.